5_databases/week2/homework_2 part 3.sql

```diff
@@ -0,0 +1,36 @@
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+use hyf_lesson1;
+
+SELECT *
+FROM task
```

Comment:
task table has no user_id. you should start joining from user_task table. this should be how the database looks like no? https://github.com/HackYourFuture-CPH/databases/blob/main/lesson2/lesson2-data.sql